cmd/outline-ss-server/config_example.yml

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+web:
+  servers:
+    - id: my_web_server
+      listen:
+        - "[::]:8000"
```

Comment:
Let's change the example to 127.0.0.1, since it's unsafe to expose HTTP publicly. It's inteded to have a local reverse proxy